oozie-to-airflow/mappers/shell_mapper.py

```diff
@@ -59,6 +60,7 @@ def _parse_oozie_node(self):
         arg_nodes = self.oozie_node.findall("argument")
         cmd = " ".join([cmd_node.text] + [x.text for x in arg_nodes])
         self.bash_command = el_utils.convert_el_to_jinja(cmd, quote=False)
+        self.pig_command = f"sh {shlex.quote(self.bash_command)}"
```

Comment:
Nice with shlex

Reply:
https://github.com/GoogleCloudPlatform/cloud-composer/blob/master/oozie-to-airflow/mappers/fs_mapper.py#L47
I've used this method before. ;-)